### ISSUE TITLE
More card backs

### ISSUE THREAD

**Issue by db0** [db0/godot-card-gaming#87]
Currently the game comes with one card back. I'd like to have more options. In fact I'd like this to be one of the easiest things to contribute, without necessarily needing heavy coding experience.

All that is needed to contribute a new card back is to create a new scene which is a panel and modify its contents to your hearts content. The scene will be instanced on its own as the card back so it's impossible to break.

We can also add extra code to the card back which would implement what I would **really** like to have, a shader-based card back. Imagine a perpetually-zooming mandelbrot set as a card back. Coo? Cool!

If you want to contribute your own card back, leave a comment on this issue and/or link your Pull Request to it.

If you have any idea for a new card back, suggest it here and we'll add it to the list.

- [ ] Infinitely zooming Mandelbrot

**Comment by ophoria** [comment]
For most games designers would opt to use raster graphics (PNG, JPG etc) instead of what I think you're going for here. Willl it be an option? Should be easy. 

On that topic: Ability to associate x number of card backs to a game, as some will use more than one.

**Comment by db0** [comment]
Yeah simple png or jpegs are even now possible. I just don't have any :)

**Comment by db0** [comment]
We got our first card back at https://github.com/db0/godot-card-gaming/pull/107 :D
